docs/overview.md

```diff
@@ -549,6 +586,28 @@ func main() {
 ```
 
 
+<a name="lenNil-ref"></a>
+## lenNil
```

Comment:
Why is this pops up in multiple PRs?
Can we merge this checker docs (if it's even merged, I don't remember) in a separate PR?
Can be a mistake here, for example, you're pushing your local docs for the checker that was not merged, perhaps?